core/src/test/java/com/valtech/aem/saas/core/fulltextsearch/DefaultFulltextSearchServiceTest.java

```diff
@@ -59,7 +59,7 @@ void testGetResults_failedRequestExecution() {
   @Test
   void testGetResults_responseBodyMissing() {
     when(searchRequestExecutorService.execute(any(SearchRequest.class))).thenReturn(
-        Optional.of(new SearchResponse(new JsonObject())));
+        Optional.of(new SearchResponse(new JsonObject(), true)));
     FulltextSearchGetRequestPayload payload = DefaultFulltextSearchRequestPayload.builder(
         new DefaultTermQuery("bar"), new DefaultLanguageQuery("de")).build();
     MatcherAssert.assertThat(testee.getResults("foo", payload).isPresent(), is(false));
```

Comment:
i notice that in some tests we use import static org.hamcrest.MatcherAssert.assertThat; and than call "assertThat" directly, i would change this so that we use it in same way in every test class (please check the other tests as well, so that i don't copy the comment multiple times)

Reply:
I agree. This seems to be an oversight. I usually add the static imports